### PR TITLE
Word wrap column set to 120

### PR DESCRIPTION
**Pull request by mebverse**

### CODE REVIEW

File: documentation-standards.rst [hunk 1]
```diff
@@ -50,9 +50,9 @@ External resources link to this documentation. Any link to a documentation refer
 about the external resource, e.g.::
 
   ..  Do not change this reference! [or document name]
-      Referred to by: tutorial message 51 project-create-base-project
-      Where: in the project creation dialog e.g. https://control.divio.com/control/project/create/#step-1
-      As: https://docs.divio.com/en/latest/background/project-creation-options#project-creation-platform
+      Referred to by: tutorial message 51 project-create-base-project Where: in the project creation dialog e.g.
+      https://control.divio.com/control/project/create/#step-1 As:
+      https://docs.divio.com/en/latest/background/project-creation-options#project-creation-platform
```
Comment (evildmp) [inline documentation-standards.rst]: These changes are incorrect.

File: project-backups.rst [hunk 2]
```diff
@@ -12,8 +12,8 @@ Our backup system is designed with pragmatic, real-world needs and concerns in m
 Backups on Divio are in accordance with two key principles.
 
 * Data integrity - the integrity of your data and content is paramount, and they must be kept safe.
-* Portability - your project belongs to you. Your data and content must be as portable as possible,
-  so that you can sign up to our services confident that you will never find yourself locked in to them.
+* Portability - your project belongs to you. Your data and content must be as portable as possible,so that you can sign
+up to our services confident that you will never find yourself locked in to them.
```
Comment (evildmp) [inline project-backups.rst]: This will break formatting of the list.

File: project-backups.rst [hunk 3]
```diff
@@ -59,9 +60,10 @@ Various policies are available but in all environments this is set by default to
 
 * Each day, the system will take a backup, and discard any daily backups older than seven days.
 * Each week, it will relabel the oldest daily backup as a weekly backup and discard any weekly backups older than one
-  month.
+month.
```
Comment (evildmp) [inline project-backups.rst]: The formatting was correct before this change.

File: security-policy.rst [hunk 4]
```diff
@@ -89,8 +91,8 @@ Security options for Divio users
 
 We’re able to add further protection for our users for their projects. For example, we offer:
 
-* free SSL certificates to all customers and HTTPS by default
-* offer security patches and layers, proactively, in response to significant CVEs
+* free SSL certificates to all customers and HTTPS by default 
+* offer security patches and layers, proactively, in response to significant CVEs
```
Comment (evildmp) [inline security-policy.rst]: This change appears to be adding whitespace to the end of the lines (it should not).

File: documentation-standards.rst (hunk 1 above)
Comment (mebverse) [reply documentation-standards.rst]: Done

File: project-backups.rst (hunk 2 above)
Comment (mebverse) [reply project-backups.rst]: done

File: project-backups.rst (hunk 3 above)
Comment (mebverse) [reply project-backups.rst]: done

**Review by mebverse — COMMENTED** [review]
I have applied the necessary changes.

File: project-backups.rst [hunk 5]
```diff
@@ -22,7 +22,8 @@ What’s backed up?
 Our backup system takes care of your project’s:
 
 * database - the database that Django uses for persistent storage, made available for download as a binary database dump
-* media files - files uploaded/processed/stored by the project’s applications, made available for download as a tarred archive, and including a manifest file of contents
+* media files - files uploaded/processed/stored by the project’s applications, made available for download as a tarred
+archive, and including a manifest file of contents
```
Comment (evildmp) [inline project-backups.rst]: I think you will need to build this to see how it appears.

**Review by evildmp — CHANGES_REQUESTED** [review]

**Pull request merged**